pkg/controllers/vdb/install_reconciler.go

```diff
@@ -224,6 +222,8 @@ func (d *InstallReconciler) generateHTTPCerts(ctx context.Context) (bool, error)
 			if err != nil {
 				return false, errors.Wrap(err, fmt.Sprintf("failed to copy %s to the pod %s", fname, p.name))
 			}
+			// Invalidate the pod facts cache since its out of date due the https generation
+			d.PFacts.Invalidate()
```

Comment:
Do you also want to invalidate podfacts here for admintools?

Reply:
It's not needed for AT because we invalidate elsewhere. But I don't think it hurts to invalidate here.